samples/boards/bbc_microbit/pong/src/main.c

```diff
@@ -483,23 +483,27 @@ void pong_remote_lost(void)
 
 static void configure_buttons(void)
 {
-	static struct gpio_callback button_cb;
+	static struct gpio_callback button_cb_data;
 	struct device *gpio;
 
 	gpio = device_get_binding(DT_ALIAS_SW0_GPIOS_CONTROLLER);
 
 	gpio_pin_configure(gpio, DT_ALIAS_SW0_GPIOS_PIN,
-			   (GPIO_DIR_IN | GPIO_INT | GPIO_INT_EDGE |
-			    GPIO_INT_ACTIVE_LOW));
+			   DT_ALIAS_SW0_GPIOS_FLAGS | GPIO_INPUT);
```

Comment:
Flags in `bbc_microbit.dts` should be also updated for this to work: `GPIO_INT_ACTIVE_LOW` -> `GPIO_ACTIVE_LOW`.

Reply:
Oh yes, well spotted! This seemed to work because the button press was being detected on release of the button, so it fooled me.